app/views/obituaries/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
     <div class="col-md-6">
       <h3 class="my-3">Obituary</h3>
-      <p class="description"><%= @obituary.description %></p>
+      <p class="description"><%= simple_format(@obituary.description)%></p>
```

Comment:
this "simple_format" was all that was needed to preserve line breaks

Reply:
Ooh I've never seen that before.

Reply:
Holy cannoli!

Reply:
Fun fact, this threw off all the tests, instead of "expect(page)" I now have to use "expect(page.html)"